borsh/src/schema.rs

```diff
@@ -903,3 +903,16 @@ impl BorshSchema for std::net::IpAddr {
         id_addr_std_derive_impl::IpAddr::declaration()
     }
 }
+
+/// Adds schema to `bytes::Bytes`.
+/// Conservatively just uses `[u8]` schema, so that any consumer already handles it.
+/// while in Rust code we can be more optimal as needed.
+#[cfg(feature = "bytes")]
```

Comment:
This feature should be declared in Cargo.toml as well.

Reply:
it probably already is declared as an implicit one from optional dependency https://github.com/near/borsh-rs/blob/master/borsh/Cargo.toml#L41